pkg/karmadactl/apply.go

```diff
@@ -180,6 +187,12 @@ func (o *CommandApplyOptions) generatePropagationObject(info *resource.Info) run
 		},
 	}
 
+	if len(o.Clusters) > 0 {
+		spec.Placement.ClusterAffinity = &policyv1alpha1.ClusterAffinity{
+			ClusterNames: o.Clusters,
```

Comment:
We should check whether the clusters exist in karmada control plane.

Reply:
@lonelyCZ done in `validate` func